### PR TITLE
Update branch xls importer, #9749

### PR DESCRIPTION
**Pull request by njkim**
### Types of changes

-   [ ] Bugfix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Description of Change

Support tileid in input data

### Issues Solved

#9749 #9748

### Checklist

-   [ ] Unit tests pass locally with my changes
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [ ] I have added necessary documentation (if appropriate)

#### Ticket Background
*   Sponsored by: 
*   Found by: @ 
*   Tested by: @ 
*   Designed by: @

### CODE REVIEW

**Review by chiatt — CHANGES_REQUESTED** [review]
This is looking good Namjun. Just a few comments that would tighten things up a bit.

File: arches/app/etl_modules/branch_excel_exporter.py [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+        )
+        return {"success": True, "data": graphs}
+
+    def get_node_lookup_by_id(self, nodes):
+        lookup = {}
+        for node in nodes:
+            lookup[str(node.nodeid)] = {"alias": str(node.alias), "datatype": node.datatype, "config": node.config}
+        return lookup
+
+    def dictfetchall(self, cursor):
```
[the remark this answers is not in the document]
Comment (njkim) [reply arches/app/etl_modules/branch_excel_exporter.py:71]: good point!

File: arches/app/etl_modules/branch_excel_exporter.py [hunk 2]
```diff
@@ -0,0 +1,229 @@
+from datetime import datetime
+from io import BytesIO
+import json
+import logging
+import os
+import zipfile
+from openpyxl.writer.excel import save_virtual_workbook
+from django.core.files import File as DjangoFile
+from django.http import HttpResponse
```
[the remark this answers is not in the document]
Comment (njkim) [reply arches/app/etl_modules/branch_excel_exporter.py]: Removed

File: arches/app/media/js/views/components/etl_modules/branch-excel-exporter.js [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+            this.resourceids = ko.observable();
+    
+            this.selectedLoadEvent = params.selectedLoadEvent || ko.observable();
+            this.validationErrors = params.validationErrors || ko.observable();
+            this.validated = params.validated || ko.observable();
+            this.getErrorReport = params.getErrorReport;
+            this.getNodeError = params.getNodeError;
+            this.alert = params.alert;
+
+            this.getGraphs = async function(){
```
[the remark this answers is not in the document]
Comment (njkim) [reply arches/app/media/js/views/components/etl_modules/branch-excel-exporter.js]: Good suggestion!

File: arches/app/etl_modules/branch_excel_exporter.py [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+"""
+
+class BranchExcelExporter(BranchCsvImporter):
+    def __init__(self, request=None, loadid=None):
+        self.request = request if request else None
+        self.userid = request.user.id if request else None
+        self.moduleid = request.POST.get("module") if request else None
+        self.loadid = loadid if loadid else None
+
+    def get_graphs(self, request):
```
[the remark this answers is not in the document]
Comment (njkim) [reply arches/app/etl_modules/branch_excel_exporter.py]: done

File: arches/app/etl_modules/branch_excel_exporter.py [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+import logging
+import os
+import zipfile
+from openpyxl.writer.excel import save_virtual_workbook
+from django.core.files import File as DjangoFile
+from django.http import HttpResponse
+from django.db import connection
+from django.utils.translation import ugettext as _
+from arches.app.etl_modules.branch_csv_importer import BranchCsvImporter
+from arches.app.models.models import GraphModel, Node, File, TempFile
```
[the remark this answers is not in the document]
Comment (njkim) [reply arches/app/etl_modules/branch_excel_exporter.py]: done

File: arches/app/models/migrations/9748_branch_excel_exporter.py [hunk 6]
```diff
@@ -0,0 +1,543 @@
+# Generated by Django 3.2.19 on 2023-07-24 18:55
+
+from django.db import migrations, models
+import django.utils.timezone
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('models', '9746_related_resource_post_save_bug'),
```
Comment (chiatt) [inline arches/app/models/migrations/9748_branch_excel_exporter.py]: This dependency now needs to change to:  `9744_add_tile_excel_importer`

**Review by chiatt — APPROVED** [review]

**Pull request merged**